core/api/v1/keytransparency.proto

```diff
@@ -92,59 +92,37 @@ message MapperMetadata {
   int64 highest_fully_completed_seq = 1;
 }
 
-// GetEntryRequest for a user object.
-message GetEntryRequest {
+// Gets the leaf entry for a user.
+message GetUserRequest {
   // directory_id identifies the directory in which the user lives.
-  string directory_id = 4;
-  // user_id is the user identifier, the format of which is defined by the
-  // application.
-  string user_id = 1;
-  // Used to be app_id. Applications can define their own hierarchy for user_id
-  // if desired.
-  reserved 2;
+  string directory_id = 1;
+  // user_id is the user identifier, the format for which is defined by the application.
+  string user_id = 2;
   // first_tree_size is the tree_size of the currently trusted log root.
   // Omitting this field will omit the log consistency proof from the response.
   int64 first_tree_size = 3;
 }
 
-// GetEntryResponse returns a requested user entry.
-message GetEntryResponse {
-  //
-  // Privacy layer hides user_id and profile data until requested.
-  //
-
-  // vrf_proof is the proof for VRF on user_id.
+// Leaf entry for a user.
+message MapLeafEntry {
+  // vrf_proof is the proof for the VRF on user_id.
   bytes vrf_proof = 1;
+  // map_inclusion is an inclusion proof for the map leaf in an accompanying
+  // trillian.SignedMapRoot. If the leaf is non-empty, its leaf.leaf_value
+  // stores a serialized Entry proto.
+  trillian.MapLeafInclusion map_inclusion = 2;
+  // committed contains the data and nonce used to make a cryptographic
```

Comment:
FYI committed is serialized inside of map_inclusion.map_leaf.extra_data, and is deserialized here for convenience.